keyboards/xd75/keymaps/skewwhiffy/config.h

```diff
@@ -19,6 +19,4 @@
 
 #include "../../config.h"
```

Comment:
The file `keyboards/xd75/keymaps/skewwhiffy/config.h` can be safely deleted, as nothing in the file has any effect on the build.

Reply:
Done.